### PR TITLE
to add new template for CET - Diploma certificate - normal

### PR DESCRIPTION
**Pull request by RP-OIS-SK**
Add new template for CET - diploma - normal 
Fix - add FIN field to separate FIN from NRIC 
---
name: Certificate Template Addition
about: This is the workflow for requesting new certificate templates to be added to
  the OpenCerts repository
title: "[New Template]"
labels: new template
assignees: ''

---

# Pull Request Guidelines for Adding Certificate Templates
This document is a work in progress but here are some basic checks. As these are only basic guidelines, meeting the below doesn't indicate there will be no issues with your pull request.

### Certificate Template 
- [X] No more than 5 templates or 25 added/modified files in the pull request
- [X] Integration test for each template that checks that the correct rendering is done given a sample certificate
- [X] Sample certificate file included for each template, located alongside the integration test for each template
- [X] Sample certificates must obviously be a sample certificate
  - [X] Obviously fictitious name
  - [X] Obviously sample signatory images
- [X] No fixed-size raster images as part of certificate layout
- [X] Mobile responsive design
- [X] Date parsing should be localised to template author's timezone
- [X] Webpack chunking code is correct
  - [X] Has chunking code
  - [X] Same chunking code as the other certificates belonging to that institute
- [X] Certificate Store Addresses have been updated
  - [X] Template Whitelist
  - [X] Registry
- [X] Template should not be using resources(images etc.) on the website outside of their own folder (e.g institute logo shouldn't be used from /static because there's no guarantee it will not change)

### Pre-merge checks

- [X] Ensure that your code has been [rebased](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) on top of latest OpenCerts master
- [X] Linter issues resolved (Run `npm run lint:fix` to see issues)
- [X] `npm run test` passes
- [X] `npm run test:integration` passes
- [X] [Travis Build passes](https://docs.travis-ci.com/user/for-beginners/)

### CODE REVIEW

**Comment by RP-OIS-SK** [comment]
Hi Raymond, 

There are some error which I do not get previously.  Please advise how to fix it.  Thanks.  


/home/travis/build/OpenCerts/opencerts-website/src/components/CertificateTemplates/tlds/sg/edu/index.js
  8:45  error  Replace `·...singaporetech,·...np,·...nyp,·...rp·` with `⏎··...singaporetech,⏎··...np,⏎··...nyp,⏎··...rp⏎`  prettier/prettier
/home/travis/build/OpenCerts/opencerts-website/src/components/CertificateTemplates/tlds/sg/edu/rp/2018-C-DCN/index.js
  2:10  error  MultiCertificateRenderer not found in 'template-utils/MultiCertificateRenderer'  import/named

**Comment by rjchow** [comment]
please fix code duplication issues

File: src/components/CertificateTemplates/tlds/sg/edu/rp/2018-C-DCN/RP_DC_15060278.opencert [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        "courseCredit": "9a0d75f5-e0bd-4eb2-bce1-75ea0ee4e2e9:string:4",
+        "courseCode": "7c7c5b84-1f0f-4cf6-8a3d-c2104944b4b9:string:E3019C",
+        "semester": "f2cb6e44-7d12-404f-b2dc-75b6502dcb28:string:AY2017 Term 2"
+      }
+    ],
+    "cumulativeScore": "ecb48ef0-c6c6-4715-99a6-857c9eacf2f5:number:3.88",
+    "additionalData": {
+      "additionalCertId": "30e0f4e1-1bbc-4acc-803f-892eddfd61bd:string:RPCETC17/05854",
+      "merit": "dc80b73d-e3b9-418c-aeac-8fdb1c6bfdfb:string:N",
+      "studentId": "6f0ef477-1414-4ea6-bfde-0774b650cbab:string:15060278",
```
Comment (rjchow) [inline src/components/CertificateTemplates/tlds/sg/edu/rp/2018-C-DCN/RP_DC_15060278.opencert]: only one minor issue, please move studentId to the recipient object

**Comment by RP-OIS-SK** [comment]
Hi Raymond,

The change is last minute and we have already completed all test.  It will
impact the issue of OpenCerts to graduates.  Can we implement it for the
next batch when we implement schema 1.5

Thanks.

SK.

**Comment by rjchow** [comment]
schema 1.5 or not, studentId belongs to the recipient object - it's a minor change please do it quickly and we can all get on with our lives

**Comment by rjchow** [comment]
ok nevermind it is quite irrelevant to anyone else i suppose
by the way I don't see your production certificate store address in the PR - please add it in or students will not be able to verify on production

**Pull request closed without merging**